packages/channels/src/twilio/renderers/location.ts

```diff
@@ -0,0 +1,11 @@
+import { LocationRenderer } from '../../base/renderers/location'
+import { LocationContent } from '../../content/types'
+import { TwilioContext } from '../context'
+
+export class TwilioLocationRenderer extends LocationRenderer {
+  renderLocation(context: TwilioContext, payload: LocationContent) {
+    const googleMapsLink = `https://www.google.com/maps/search/?api=1&query=${payload.latitude},${payload.longitude}`
+
+    context.messages.push({ body: `${payload.title}\n\n${googleMapsLink}` })
+  }
```

Comment:
We can't send location with SMS but we can send this link that points to the location in google maps. On my android phone it event displays the location in my SMS app. Not sure about behavior for iPhones but it's always possible to click the link